Makefile

```diff
@@ -0,0 +1,18 @@
+# Docker is the only dependency here
+# Build the dashboard image for host or cross-compile
+# This Makefile can only be run on a amd64 host
+
+build-image:
```

Comment:
Could we keep `golang` as the base? It's a debian image with some essential addons
Otherwise, I'll simply copy over the packages that are required from these `Dockerfiles`

Here's the chain: 
- `golang:1.5.3` adds `go` of course. Based on `buildpack-deps:jessie-scm
- `buildpack-deps:jessie-scm` adds `git` and such things. Based on `buildpack-deps:jessie-curl`
- `buildpack-deps:jessie-curl` adds `curl` and `ca-certificates`. Based on `debian:jessie`